pkg/plugin/manager/manager.go

```diff
@@ -67,6 +67,10 @@ func (m *manager) setDB() error {
 }
 
 func (m *manager) RunMigrations() error {
+	if len(m.GenConfig.Schema) <= 0 {
+		return fmt.Errorf("config is missing a schema, required for plugin migrations")
```

Comment:
nitpick but can use `errors.New("")` if not formatting an error

Reply:
👍